fetcher/nvd/json/nvd.go

```diff
@@ -176,6 +182,10 @@ type CveItem struct {
 	LastModifiedDate string `json:"lastModifiedDate"`
 }
 
+type CertLink struct {
```

Comment:
__[Golint]__ exported type CertLink should have comment or be unexported ([link](https://sider.review/gh/repos/54628930/pulls/129#issue-4304149))

<sub>posted by Sider</sub>